dhcpv4/option_ipv6_only_preferred.go

```diff
@@ -0,0 +1,10 @@
+package dhcpv4
+
+import (
+	"time"
+)
+
+// The IPv6-Only Preferred option is described by RFC 8925, Section 3.1
+func OptIPv6OnlyPreferred(d time.Duration) Option {
```

Comment:
Maybe we should add this option to the same file where the type `Duration` is defined and rename that file to a more appropriate name.

Reply:
Merged as `option_duration.go`